src/tbitfield.cpp

```diff
@@ -9,30 +9,49 @@
 #include <iostream>
 #include "tbitfield.h"
 
-TBitField::TBitField(size_t len)
-{
 
+
+size_t bitLen = 0;   // длина битового поля - макс. к-во битов
+elType* pMem = 0;      // память для представления битового поля
```

Comment:
use `nullptr`
dublicated in `include/tbitfield.h`

Reply:
fixed nullptr